packages/itwinui-variables/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@itwin/itwinui-variables",
+  "version": "0.1.0",
+  "author": "Bentley Systems",
+  "license": "MIT",
+  "main": "index.css",
+  "style": "index.css",
+  "files": [
+    "index.css",
+    "LICENSE.md"
+  ],
+  "description": "CSS variables for the iTwinUI design system",
+  "homepage": "https://github.com/iTwin/iTwinUI",
+  "keywords": [
+    "css",
+    "theme",
+    "itwinui",
+    "itwin",
+    "design system",
+    "design token"
+  ],
+  "dependencies": {},
+  "devDependencies": {
+    "autoprefixer": "^10.4.7",
+    "postcss": "^8.4.14",
+    "sass": "^1.52.2"
```

Comment:
Not sass-embeded?

Reply:
sass-embedded doesn't seem to have a cli, and it's only file so performance is not an issue